pkg/eosclient/eosbinary/eosbinary.go

```diff
@@ -207,6 +208,8 @@ func (c *Client) executeXRDCopy(ctx context.Context, cmdArgs []string) (string,
 				err = nil
 			case int(syscall.ENOENT):
 				err = errtypes.NotFound(errBuf.String())
+			case 54:
```

Comment:
This is all good but unfortunately xrootd may return `54` for several errors as far as I know (including on conditions that could be retried). We could add a log that includes the full `errBuf` to start with...